### PR TITLE
Organization Details Component

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Adds the organization details component, a delete org button component, and a delete org modal component. When the user selects the delete org button the delete org modal component should appear with info on how to delete an org.

Note: I noticed that the blue bars at the top of the organization details component moves to the left when the button appears. I tried a few different things, but it always moves. 

I tried using a modal instead of a button and that removes the issue with the blue bars moving. Here's a [recording](https://www.awesomescreenshot.com/video/14654544?key=237ae20e59c4f281fb96336e000dfd2d). However, this is different from the design and the modal's background color wouldn't change.  

If we can't figure out how to stop the blue bars from moving, we may have to discuss a small change to the design with Edwin. 

Fixes SC-12520

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/14654808?key=a0ffea13ba43fd4925507860a948f805

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/organizations/components/OrganizationDetails/OrganizationDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <section className="mt-8 pl-6 max-w-4xl border-2 border-secondary-500 rounded-md">
+            <div className="mt-4 absolute right-28">
+                <BlueBars onClick={handleOpen} />
+                <div className="relative left-12">
+                    {showButton && <DeleteOrg close={handleClose} />}
+                </div>    
+            </div>
+            <div className="flex py-8 gap-4">
+                <h6 className="font-bold">Organization Name:</h6>
+                <span>Name</span>
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/organizations/components/OrganizationDetails/OrganizationDetails.tsx:24]: This span and the other spans were used as placeholders to help me view the style. I'll replace these while working on the organization data table component.

File: web/beacon-app/src/features/organizations/components/DeleteOrgModal/DeleteOrgModal.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+export default function DeleteOrgModal(props: any) {
+  return (
+    <div>
+      <Modal
+        title="Delete Organization"
+        open={true}
+        containerClassName="max-w-md"
+      >
+        <Fragment key=".0">
```
Comment (masskoder) [inline web/beacon-app/src/features/organizations/components/DeleteOrgModal/DeleteOrgModal.tsx:14]: Where this fragment key came from?

**Review by masskoder — APPROVED** [review]
Looks good.
It would be great if you can add the public/tailwind.css in the .gitignore. we don't need to push it

File: web/beacon-app/src/features/organizations/components/DeleteOrgModal/DeleteOrgModal.tsx (hunk 2 above)
Comment (daniellemaxwell) [reply web/beacon-app/src/features/organizations/components/DeleteOrgModal/DeleteOrgModal.tsx:14]: It's listed in Storybook for the modal component. I kept getting an error message without it, so I added it. Below is a screenshot from the Storybook docs.

Screenshot 2023-02-09 at 10 20 07 AM (2)

**Comment by daniellemaxwell** [comment]
@masskoder The public tailwind.css file is in the .gitignore, but it wasn't ignored for some reason. I'll remove it before merging to main.

**Pull request merged**